### PR TITLE
NOX-703 Dotnet 8 upgrade

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

**Comment by rochar** [comment]
H @msadikkoseaupa 
please do not merge without the 3 approvals.
build is failing, please also fix it for the review
thks

**Review by rochar — APPROVED** [review]

**Review by jan-schutte — APPROVED** [review]
Looks good

**Pull request merged**
